app/assets/javascripts/pages.js.coffee

```diff
@@ -1,3 +1,17 @@
+@['pages#sponsor'] = (data) -> 
+  handler = Gmaps.build('Google')
+  handler.buildMap
+    provider:
+      center:
+        lat: 39
+        lng: -101
+      zoom: 3
+      scrollwheel: false
+    internal:
+      id: 'sponsor-map-jumbotron'
+    ->
+      handler.addMarkers markersJSON
+
```

Comment:
Can you make this addition to the bottom of the file? It's a clearer logical progression to have `pages#home` at the beginning.